Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Choose alpine base image
+FROM node:alpine3.17
+# Set working directory
+WORKDIR /usr/src/app
+# Copy package.json and yarn.lock to working directory
+COPY package.json yarn.lock ./
+# Install dependencies
+RUN yarn install
+# Copy ARG to ENV
+ARG NODE_ARG
+# Set docker ARG to ENV
+ENV NODE_ENV $NODE_ARG
+# Copy all files to working directory
+COPY . .
+# Build the app
+RUN yarn build
```

Comment:
I don't think you need this if you're running `yarn serve`. `yarn build` builds so it can be served via a static web server.  `yarn serve` will just build it at start up every time anyway.